hist/hist/inc/TGraph.h

```diff
@@ -178,7 +178,7 @@ class TGraph : public TNamed, public TAttLine, public TAttFill, public TAttMarke
    virtual Int_t         RemovePoint(); // *MENU*
    virtual Int_t         RemovePoint(Int_t ipoint);
    void                  SavePrimitive(std::ostream &out, Option_t *option = "") override;
-   void                  SaveAs(const char *filename, Option_t *option = "") const override; // *MENU*
+   void                  SaveAs(const char *filename = "graph", Option_t *option = "") const override; // *MENU*
```

Comment:
Can we keep the changes of this PR related to `TH1` only so we address the related issue with more focus? Since this default argument was simply untested, we would need some extra tests also for `TGraph`, but I think this should go in another PR

Reply:
yep!